src/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- fonts -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Kalam:wght@300;400&display=swap" rel="stylesheet">
+<!-- fontawesome -->
+  <script src="https://kit.fontawesome.com/1bc3a7e061.js" crossorigin="anonymous"></script>
+  <title>Document</title>
+</head>
+<body>
+  <main class="container">
+    <!-- <header class="title-section"> -->
+      <h4>Today's To Do </h4> <i class="fa-solid fa-arrows-rotate" aria-label="Rotate icon"></i>
+    </header>
+    <section class="form-field-section">
+      <form action="submit" class="input-field">
+        <input type="text" id="input-task" class="input-task" autocomplete="off" required>
+        <label for="input-task">Add your task here...</label>
+        <input type="button" id="addBtn" value="+">
+      </form>
```

Comment:
Would please update the input tag inside the form to have type="submit" instead of type="button". This change is more semantically correct, as the button's purpose is to submit the form.